cnf-certification-test/identifiers/identifiers.go

```diff
@@ -663,6 +669,15 @@ var Catalog = map[claim.Identifier]TestCaseDescription{
 		BestPracticeReference: bestPracticeDocV1dot3URL + " Section 5.3.7",
 		ExceptionProcess:      NoDocumentedProcess,
 	},
+	TestContainerIsCertifiedDigestIdentifier: {
```

Comment:
Did you want to try to use the new `AddCatalogEntry()` func for this test description?  It works either way but we've been trying to add new test cases to the catalog via that func.